src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -181,9 +230,27 @@ const PathwaysListElement: FC<PathwaysListElementProps> = ({
                 ))}
               </tbody>
             </table>
-            <button className={indexStyles.button} onClick={(): void => callback(evaluatedPathway)}>
+            <button
```

Comment:
This button should have consistent styles to our other buttons. I would add `className={indexStyles.button}` to this button element and then add this to `src/styles/button.scss`:
```js
.button {
  ...

  &:disabled {
    background-color: lighten($blue, 20%);
    cursor: default;
  }
}
```